plugins/portals-list.js

```diff
@@ -274,19 +332,19 @@ window.plugin.portalslist.portalTable = function(sortBy, sortOrder, filter) {
 
   var length = window.plugin.portalslist.listPortals.length;
 
-  ["All", "Neutral", "Resistance", "Enlightened"].forEach(function(label, i) {
+  ["All", "Neutral", "Resistance", "Enlightened", "visited", "captured", "Sc. contr." ].forEach(function(label, i) {
```

Comment:
```suggestion
  ["All", "Neutral", "Resistance", "Enlightened", "Visited", "Captured", "Sc. contr." ].forEach(function(label, i) {
```

Reply:
> "Sc. contr."

I don't like

Reply:
Agree, but anything else explodes the field.